### ISSUE TITLE
Incorrect file name on o365beat-1.4.3-x86_64.rpm.sha512

### ISSUE THREAD

**Issue by fuef** [counteractive/o365beat#28]
```
$ sha512sum -c o365beat-1.4.3-x86_64.rpm.sha512 
sha512sum: o365beat-7.4.0-x86_64.rpm: No such file or directory
o365beat-7.4.0-x86_64.rpm: FAILED open or read
sha512sum: WARNING: 1 listed file could not be read
```

Apparently, the file name should read o365beat-**1.4.3**-x86_64.rpm

**Comment by chris-counteractive** [comment]
Absolutely, this is good feedback.  This is related to the packaging issue in #10, where the build process (`make release`) uses the libbeat version (in this case 7.4.0) for some things and the beat version (in this case 1.4.3) for others.  I'll look into it more and see if there's a fix that doesn't need upstream changes to libbeat, otherwise I'll create an issue with them so we can track it here.

Thanks for the issue.
